### ISSUE TITLE
iOS: add “Refresh” command to App Context Menu

### ISSUE THREAD

**Issue by kevinm6** [Ranchero-Software/NetNewsWire#3022]
It could be useful, like on macOS, had "refresh" command on NNW context menu in Home Screen, that okay open the app but invoke even the refresh of all the feeds (also like Apple Podcast app).

![image](https://user-images.githubusercontent.com/72861758/115233099-c9fc3180-a117-11eb-87fa-d92234b87ba8.jpeg)

**Comment by stuartbreckenridge** [comment]
The app will refresh itself on opening if it's been in the background for at least 15 minutes, if I recall correctly.

**Comment by kevinm6** [comment]
Yes, usually it updates itself after an amount of time, but if I open it before that amount, I have to manually pull to refresh the feeds, and usually with many Web Feeds, several have new articles!
Maybe is just for me, but I think it could be a useful addition, even because Apple Podcast app has it even with setting of time for refresh the episodes.
